app/models/memberships/join_zusatzsektion.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+module Memberships
+  class JoinZusatzsektion < MemberJoinSectionBase
+
+    def initialize(join_section, person, join_date, family_membership: false, **params)
+      super(join_section, person, join_date, **params)
+      @family_membership = family_membership
+      @group = join_section.group_for_neuanmeldung
```

Comment:
```suggestion
      @neuanmeldungen_group = join_section.group_for_neuanmeldung
```

etwas längerer name, dafür aber gleich klar um was es sich handelt ?!

Reply:
via delegation gelöst